#!/bin/sh /usr/share/dpatch/dpatch-run
## 10-no-registration.dpatch by Michael Meskes <meskes@debian.org>
##
## DP: Do not ask for registration

@DPATCH@

diff -Naurp virtualbox-ose-1.5.2-dfsg2.orig/src/VBox/Frontends/VirtualBox/Makefile.kmk virtualbox-ose-1.5.2-dfsg2/src/VBox/Frontends/VirtualBox/Makefile.kmk
--- virtualbox-ose-1.5.2-dfsg2.orig/src/VBox/Frontends/VirtualBox/Makefile.kmk	2007-10-18 11:07:28.000000000 +0000
+++ virtualbox-ose-1.5.2-dfsg2/src/VBox/Frontends/VirtualBox/Makefile.kmk	2007-12-18 14:53:06.000000000 +0000
@@ -38,9 +38,9 @@ include $(PATH_KBUILD)/header.kmk
 #
 # exclude inappropriate UI content
 #
-#ifndef VBOX_OSE
+ifndef VBOX_OSE
 VBOX_WITH_REGISTRATION := 1
-#endif
+endif
 
 ifndef VBOX_WITH_REGISTRATION
 VirtualBox_QT_UISRCS := $(filter-out ui/VBoxRegistrationDlg.ui,$(VirtualBox_QT_UISRCS))
